src/static/js/main.js

```diff
@@ -19,3 +19,11 @@ $( '.feature-list' ).append(
   'cf-components.</p>' +
   '</section>'
 );
```

Comment:
You need to remove this obsolete `feature-list` counting stuff. It's throwing a console error and stopping the execution of the new script that you added below.